### PR TITLE
feat: completed poll event config

### LINKED ISSUE #211: Polls settings page

### PR DESCRIPTION
**Pull request by chungwwei**
Closes #211

### CODE REVIEW

File: lib/models/messages/twitch/event.dart [hunk 1]
```diff
@@ -100,7 +106,9 @@ class TwitchPollEventModel extends MessageModel {
         pollTitle: data['event']['title'],
         isCompleted: true,
         messageId: "poll${data['event']['id']}",
-        timestamp: data['timestamp'].toDate());
+        timestamp: data['timestamp'].toDate(),
+        startTimestamp: data['event']['started _at'].toDate(),
```
Comment (juaoose) [inline lib/models/messages/twitch/event.dart:110]: `started _at` there is an extra space.

File: functions/package.json [hunk 2]
```diff
@@ -31,4 +31,4 @@
     "typescript": "^4.3.5"
   },
   "private": true
-}
+}
```
Comment (juaoose) [inline functions/package.json:34]: No real change here I guess?

File: lib/models/messages/twitch/event.dart [hunk 3]
```diff
@@ -100,7 +106,9 @@ class TwitchPollEventModel extends MessageModel {
...
+        endTimestamp: data['event']['ended_at'].toDate());
```
Comment (juaoose) [inline lib/models/messages/twitch/event.dart:111]: I guess the only thing that we might want to test with some data is if pin behaviour should change depending on the `status` property, maybe we don't want to pin in on of the two other status (archived and/or terminated)

**Review by juaoose — CHANGES_REQUESTED** [review]

File: lib/models/messages/twitch/event.dart (hunk 3 above)
Comment (chungwwei) [reply lib/models/messages/twitch/event.dart:111]: I will make it auto unpin when the poll status is `archived` or `terminated`.

**Review by juaoose — APPROVED** [review]

**Pull request merged**